bin/_gaffer.py

```diff
@@ -76,6 +76,16 @@ def prependToPath( pathToPrepend, envVar ) :
 
 gafferRoot = pathlib.Path( __file__ ).resolve().parents[1]
 
+# Clean environment setup
+# =======================
+
+# Not every environment variable we set for Gaffer itself is something
+# we want to be inherited by subprocesses that Gaffer launches. Store
+# clean copies that can be restored by `__gaffer.py`.
+
+if sys.platform == "linux" :
+	os.environ["__GAFFER_RESTORE_LD_PRELOAD"] = os.environ.get( "LD_PRELOAD", "" )
```

Comment:
When launched from an environment with `LD_PRELOAD` unset, this results in the clean environment containing `LD_PRELOAD=""`. Functionally equivalent, but not quite the same environment. Not sure how much we care about that in practice.

Reply:
Pushed https://github.com/GafferHQ/gaffer/pull/6645/commits/9696dd85c7bf0c6203df46ea7e8077d77baf5780 to remove the env var in this case.